src/lib/blockchain/wallet.ts

```diff
@@ -211,7 +211,7 @@ export class Wallet {
           symbol: "MATIC",
           decimals: 18,
         },
-        rpcUrls: ["https://matic-mumbai.chainstacklabs.com"],
+        rpcUrls: ["https://rpc.ankr.com/polygon_mumbai"],
```

Comment:
It would be nice if we had 3-4 fallback URLs to cycle through ;)

Reply:
This is a good idea 👍